webpack.config.babel.js

```diff
@@ -99,6 +100,14 @@ module.exports = {
         new webpack.ProvidePlugin({
             [options.jsxPragma]: path.join(__dirname, "utils", "element"),
         }),
+        // Copies the extension configuration and other extension assets to the build folder.
+        new CopyWebpackPlugin([
+        {
+            from: "./extension-config",
+            to: "./",
+            ignore: [ ".eslintrc" ]
+        }
+        ])
```

Comment:
Nice, is the extension different between prod and dev builds? Do we need to do anything special between dev and prod?